lms/static/js/learner_dashboard/program_list_factory.js

```diff
@@ -5,20 +5,38 @@
         'js/learner_dashboard/views/collection_list_view',
         'js/learner_dashboard/views/sidebar_view',
         'js/learner_dashboard/views/program_card_view',
-        'js/learner_dashboard/collections/program_collection'
-    ],
-    function (CollectionListView, SidebarView, ProgramCardView, ProgramCollection) {
-        return function (options) {
-            new CollectionListView({
-                el: '.program-cards-container',
-                childView: ProgramCardView,
-                collection: new ProgramCollection(options.programsData)
-            }).render();
+        'js/learner_dashboard/collections/program_collection',
+        'js/learner_dashboard/views/certificate_view',
+        'js/learner_dashboard/collections/certificate_collection'
+        ],
+        function (
+            CollectionListView,
+            SidebarView,
+            ProgramCardView,
+            ProgramCollection,
+            CertificateView,
+            CertificateCollection
+            ) {
+            return function (options) {
+                new CollectionListView({
+                    el: '.program-cards-container',
+                    childView: ProgramCardView,
+                    collection: new ProgramCollection(options.programsData)
+                }).render();
 
-            new SidebarView({
-                el: '.sidebar',
-                context: options
-            }).render();
-        };
-    });
+                new SidebarView({
+                    el: '.sidebar',
+                    context: options
+                }).render();
+
+                if (options.certificatesData.length > 0) {
+                    new CollectionListView({
+                        el: '.certificate',
+                        childView: CertificateView,
+                        collection: new CertificateCollection(options.certificatesData)
+                    }).render();
```

Comment:
There should be a CertificateView which contains this code but also adds the content as I mention in the template below. This is to keep the views as self-contained as possible and enable easy reusability.